spec/features/organizers/teams/edit_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'Editing Teams', type: :feature do
+  let(:user)            { create(:organizer) }
+  let(:season)          { Season.current }
+  let(:other_season)    { create(:season) }
+  let(:team)            { create(:team, name: 'Team Yay', season: season) }
+  let(:selection_phase) { season.applications_close_at + 1.day }
+  let!(:exercism)       { create(:project, :accepted, name: 'Exercism', season: season) }
+
+  before do
+    create(:project, :accepted, name: 'Some Old Project', season: other_season)
+    create(:project, name: 'Non-Accepted', season: season)
+    create(:project, :accepted, name: 'Open Source Dropbox', season: season)
+
```

Comment:
Not a review but an inquiry, to get better at writing specs: I'd think that you can use `build_stubbed` here instead of `create`? Just ignore me if that's incorrect.

Reply:
It's super rare that one needs or can use `build_stubbed` - I personally wouldn't even include it into `FactoryBot` if I were the author 😉 

Here, since we are in a "feature" spec which uses the app _"end to end"_ the controller is pulling stuff out of the database so things actually need to be persisted - therefore the `create`.

Reply:
O thanks for the great explanation! And I'll stop looking at uses for build_stubbed starting NOW.

Reply:
Haha, well it probably has its usecases somehow. But I usually could archive anything I wanted with `build` alone.